AGENTS.md

```
@@ -155,4 +155,31 @@ pub fn get_all_tips() -> Result<(), GetAllTipsError> {
+## Commit Guidelines
+
+* A commit has five parts: type, scope, description, body, and footer.
+* Type: feat, fix, docs, refactor, perf, style, test, chore, ci, revert, build.
+* Scope: The name of the feature or module being modified.
+* Description: A brief description of the change.
+* Body: A detailed description of the change. Start with the motivation for the change and then list the changes made.
+* Footer: Any additional information about the change.
+
+Example:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)